src/LibraryPage.tsx

```diff
@@ -1,9 +1,81 @@
-import React from "react";
-import { useParams } from "react-router-dom";
+import React, { Component } from 'react';
+import { render } from 'react-dom';
+import './style.css';
+import { Card } from "react-bootstrap"
+import {Button} from "react-bootstrap"
 
-const LibraryPage: React.FC = () => {
-  const { id } = useParams();
-  return <span>User ID: {id}</span>;
-};
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      name: 'React'
+    };
+  }
 
-export default LibraryPage;
+  render() {
+    return (
+      <div>
+ <Card style={{ width: '18rem' }}>
```

Comment:
Is a style override like this necessary? Will this work for all phone sizes? Also it might be better to use create a new component via styled components via something like (though I'm not positive it will work with boostrap components)
``const MyCard = styled(Card)` ... my custom css` ``
https://styled-components.com/docs/basics#extending-styles